bin/setup

```diff
@@ -16,16 +15,13 @@ FileUtils.chdir APP_ROOT do
   # Add necessary setup steps to this file.
 
   puts "== Installing dependencies =="
-  system! "gem install bundler --conservative"
   system("bundle check") || system!("bundle install")
 
-  # This line isn't present in Rails 7 default
   # Install JavaScript dependencies
-  system! "bin/yarn"
-
+  system("yarn install --check-files")
```

Comment:
from https://github.com/rails/rails/blob/624fe3cdb9ab774ff598af29f408425178da6677/railties/lib/rails/generators/rails/app/templates/bin/setup.tt#L19

maybe we can delete `bin/yarn` now